### ISSUE TITLE
Connection Issue with no wifi client

### ISSUE THREAD

**Issue by jaywax** [domogik/domodroid#158]
Device with no wifi card can't connect to the domogik server.
During the synchronisation process, the error "no connection" is comming up.

**Comment by tikismoke** [comment]
Did you still have this bug?

**Comment by jaywax** [comment]
Yes, still the same issue with my android device which doesn't have wifi card (only ethernet adaptator).

**Comment by jaywax** [comment]
Two different situations with the same symptom, same error : 
- One device plugged in lan with ethernet adaptator (no wifi, but wire.d) can't connect to DMG server.
- One device plugged via gsm (and vpn, no wifi) has the same error : can't connect to DMG server.
The common point is the no wifi connection.
Will try later a vpn connection under wifi.

**Comment by tikismoke** [comment]
Need to have a look with this:

https://stackoverflow.com/questions/22302548/how-to-know-whether-android-is-connected-to-wifi-or-ethernet

But trough GSM I don't understand why sync won't work.

**Comment by jaywax** [comment]
Hello,
I've done the following tests : 
- Under GSM (Wan IP + RDR port for MQ & REST SSL for domogik server) 
- Under Openvpn (not connected to wifi) with and without [proxy](https://github.com/domogik/domogik/issues/530) configuration for rest domogik
I've still the same error (no connection)

**Comment by jaywax** [comment]
Shame on me :)
> 
    Under GSM (Wan IP + RDR port for MQ & REST SSL for domogik server)
    Under Openvpn (not connected to wifi) with and without proxy configuration for rest domogik
    I've still the same error (no connection)
By using "external parameters" on domodroid with good configuration, i'm able to connect to my home, with gsm, ssl, proxy etc....
I will try on my wired tablet by using this configuration (by letting my wifi card unconfigured) to see.

**Comment by tikismoke** [comment]
You could test like that but it should not work like this.

External won't use MQ (not sure) and is really to use when you're domogik is not at the same address.

For example on my local WiFi I use the local up of domogik (192.....) And from external I use the good dns (Wich is forward to domogik) but with another port.

Also you could simply try with the QR code from domogik and it should work as simple as possible.
